comp-sci/networking/dns/dns-duplicity.md

```diff
@@ -0,0 +1,58 @@
+---
+author: TebbaVonMathenstein
+
+levels:
+
+  - medium
+
+
+tags:
+
+  - introduction
+
+  - workout
+
+
+type: exercise
```

Comment:
So, Exercises don't have questions, they only have Exercise Text.
This is really more of an insight, since it has questions.

Reply:
`type: normal` is what you wanna put here